test/e2e/generated/BUILD

```diff
@@ -13,6 +13,7 @@ load(
 go_library(
     name = "go_default_library",
```

Comment:
since `gobindata_util.go` now depends on `github.com/golang/glog", add 
```
deps = [
    "//vendor:github.com/golang/glog"
],
```
after the `srcs` section.